app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AssertAdminNewsletterConfigFieldGuestNotAllowedActionGroup.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminNewsletterConfigFieldGuestNotAllowedActionGroup">
```

Comment:
I'm not sure, but there's no assertion in this Action Group. Maybe there's other name that is better fit?

Reply:
Hi @lbajsarowicz .
I have applied Your recommendation. 

Thank you for your advice.